syntax = "proto3";
package person;
import "proto/date.proto";

/*
*The Person message
*/
message Personal {
    string first_name = 1; //""
    string second_name = 2;
    string linkedin_profile = 3;
    repeated string phone_numbers = 4; //[]
    string personal_website_link = 5;
    string github_link = 6;
    bool header_finished = 7;//false
    bytes profile_picture = 8;//empty
    int32 page_number = 9; //0
    Education education_level = 10;
    date.Date graduation_day = 11;
    repeated Address addresses = 12;
    Address single_address = 13;
}

enum Education {
    DEFAULT = 0;
    DIPLOMA = 1;
    BACHELORS = 2;
}

message Address {
    string address_one = 1;
    string address_two = 2;
    string postal_code = 3;
    string prefecture_name = 4;
    string ward = 5;
}